src/main/java/org/takes/rq/RqMethod.java

```diff
@@ -94,6 +96,13 @@
     final class Base extends RqWrap implements RqMethod {
 
         /**
+         * HTTP method line pattern.
+         */
+        private static final Pattern PATTERN = Pattern.compile(
+            "(GET|POST|PUT|DELETE|HEAD|OPTIONS|PATCH) [^ ]+( [^ ]+){0,1}"
```

Comment:
@bdragan there is only one space allows after uri (http://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html)
```
Request-Line   = Method SP Request-URI SP HTTP-Version CRLF
```
but the method line like the string below will be valid
```
GET http://www.w3.org/pub/WWW/TheProject.html                               HTTP/1.1
```